templates/idx.html

```diff
@@ -909,7 +911,7 @@ <h3 class="title is-3">
                     var containerPort = map[1];
                     result +=
                         '<li>Port ' + hostPort + ' -> ' + containerPort +
-                        ' (<a href="https://' + window.location.hostname + ':' + hostPort + '/" target="_blank">link</a>)</li>';
+                        ' (<a href="https://' + demoFqdn + ':' + hostPort + '/" target="_blank">link</a>)</li>';
```

Comment:
Why is this using https? What if the endpoint is plaintext HTTP?
Can we avoid specifying the protocol at all here by dropping the scheme and letting the browser handle that?

Also couldn't we avoid the `<div>` magic and use the `{{ demo_fqdn }}` directly here?

Reply:
sure i can drop the https

I would prefer not to have the templates generate anything in the script tag